### DISCUSSION TITLE
FF / Hypertrophy / Core

### DISCUSSION THREAD

**Discussion by giscus** [JirenXmoksh/FEE-II-PRJ#15] [General]
# FF / Hypertrophy / Core

http://127.0.0.1:5500/UI/FEE%20-%20II/PRJEXC/source/hyp_Abdomen.html

**Comment by JirenXmoksh** [comment]
This is the start of this thread! Feel free to discuss your opinions or share your experiences with the FITFUSION community. Happy exercising! <3
